src/Serokell/Util/Trace.hs

```diff
@@ -1,44 +0,0 @@
-{-# LANGUAGE FlexibleInstances      #-}
```

Comment:
This module shouldn't be deleted. Yet you won't see function usages from this module in project it's still good to have this module. It's used for debugging purposes.